README.md

```diff
@@ -15,45 +14,80 @@ Using maven, include it as a dependency:
     </repository>
 </repositories>
 ```
-```
+```xml
 <dependencies>
     <dependency>
         <groupId>com.github.willowtreeapps</groupId>
         <artifactId>conductor</artifactId>
-        <version>2.5.0</version>
+        <version>3.0.0</version>
```

Comment:
should update the version in the pom too

Reply:
Done.